lib/screens/alerts/alert.ex

```diff
@@ -592,4 +592,16 @@ defmodule Screens.Alerts.Alert do
 
   def direction_id(%__MODULE__{informed_entities: informed_entities}),
     do: List.first(informed_entities).direction_id
+
+  def is_child_stop_closure?(%__MODULE__{
+        effect: :station_closure,
+        informed_entities: informed_entities
+      }) do
+    affected_platform_ids =
+      Enum.reject(informed_entities, &String.starts_with?(&1.stop, "place-"))
+
+    length(affected_platform_ids) == 1
```

Comment:
I broke this logic by doing a Station Closure at Alewife - it only has one child stop so you hit this logic even though the entire station is closed.

Reply:
Honestly though, I'm not sure if I even care - this would be an incredibly rare situation and the outcome is fine (the fallback is very reasonable)

Reply:
If it solves the Alewife problem (and doesn't break things at Downtown Crossing) I'd be fine with switching back to that